release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1819,6 +1819,14 @@ As a workaround, if you have an installation configuration file, update it with
 
 * When you deploy a cluster with a Network Type of `OVNKubernetes` on {ibmpowerProductName}, compute nodes may reboot because of a kernel stack overflow. As a workaround, you can deploy the cluster with a Network Type of `OpenShiftSDN`. (link:https://issues.redhat.com/browse/RHEL-3901[*RHEL-3901*])
 
+* The following known issue applies to users who upgraded their {product-title} deployment to an early access version of {product-version} using release candidate 3 or 4: 
++
+With the introduction of the node identify feature, some pods are running as root. For users who updated to an early access version of {product-title} {product-version}, attempting to upgrade to the official {product-version} might not progress with the Network Operator reporting the following state: `DaemonSet "/openshift-network-node-identity/network-node-identity" update is rolling`. 
```

Comment:
I would rephrase this:
```
With the introduction of the node identify feature, some pods are running as root. For users who updated ...
```
to 
```
After the introduction of the node identify feature, some pods that were running as root will be updated to run unprivileged. As a consequence, for users who updated ...
```

and
```
not progress with the Network Operator reporting the following state:
```
to
```
not progress with the Network Operator reporting the following state persistently:
```

Just an idea, edit as you see fit